src/components/assets/Account.vue

```diff
@@ -240,6 +273,27 @@ export default defineComponent({
       { immediate: false }
     );
 
+    const bg_img = {
+      native: require('/src/assets/img/account_bg_native.webp'),
+      shiden: require('/src/assets/img/account_bg_shiden.webp'),
+      testnet: require('/src/assets/img/account_bg_testnet.webp'),
+      zk: require('/src/assets/img/account_bg_zk.webp'),
+      testnet_zk: require('/src/assets/img/account_bg_testnet_zk.webp'),
+    };
+
+    const bg = computed<String>(() => {
+      if (currentNetworkIdx.value === 0) {
```

Comment:
Use `endpointKey` enum instead numbers

Reply:
Fixed it in this [commit](https://github.com/AstarNetwork/astar-apps/pull/956/commits/0c5ca2f0a0b5e792786a55fe339f2ae21fcdbbe4)